addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -311,3 +315,58 @@ def _get_iap_search_term(self, email):
         """
         domain = tools.email_domain_extract(email)
         return ("@" + domain) if domain not in iap_tools._MAIL_DOMAIN_BLACKLIST else email
+
+    def _prepare_translations(self, plugin):
+        """
+        Returns translations for terms used in the mail addons, this method can be overridden by other modules to return
```

Comment:
Side remark: could be great to keep docstring under 80 or 100 characters, helps reading in diff mode :)

Reply:
changed, now lines are limited to 80 :)